docs/ref/forms/fields.txt

```diff
@@ -1228,6 +1228,16 @@ method::
 
         Same as :class:`ModelChoiceField.to_field_name`.
 
+.. method:: ModelMultipleChoiceField.validate_choices(qs, key, values)
```

Comment:
This signature doesn't match the code: `key` -> `field_name`.
Perhaps `values` would be better named `selected_values` or even `selected_choices` (given the method is called `validate_choices`.